src/utils/viewer/viewer.cpp

```diff
@@ -737,62 +587,53 @@ void Viewer::mouseMoveEvent(MouseMoveEvent& event) {
 // NOTE: Mouse + shift is to select object on the screen!!
 void Viewer::keyPressEvent(KeyEvent& event) {
```

Comment:
Do we have a help button to display keyPress events? Or print them out to terminal somehow?

Reply:
Added some help text.